Added more fields to chat preview structure

vkjr opened the pull request:
ChatPreview structure was extended to pass more data to mobile needed for generating the last message preview.
[Related mobile issue](https://github.com/status-im/status-mobile/issues/15281).

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [x] Have you tested changes with mobile?
- [x] Have you tested changes with desktop?

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (65)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | e7383adb | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/1/) | 2023-04-16 18:23:03 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230416-181946-e7383a-pr3397.zip) |
| :heavy_check_mark: | e7383adb | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/1/) | 2023-04-16 18:24:22 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230416-181946-e7383a-pr3397.zip) |
| :heavy_check_mark: | e7383adb | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/1/) | 2023-04-16 18:24:48 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230416-181944-e7383a-pr3397.aar) |
| :heavy_check_mark: | e7383adb | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/1/) | 2023-04-16 18:32:56 | ~13 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f253a56c | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/2/) | 2023-04-20 10:53:43 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230420-105058-f253a5-pr3397.zip) |
| :heavy_check_mark: | f253a56c | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/2/) | 2023-04-20 10:54:42 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230420-105054-f253a5-pr3397.zip) |
| :heavy_check_mark: | f253a56c | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/2/) | 2023-04-20 10:55:24 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230420-105054-f253a5-pr3397.aar) |
| :heavy_multiplication_x: | f253a56c | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/2/) | 2023-04-20 11:17:08 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/2//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | ef10aa06 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/3/) | 2023-04-23 16:17:24 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/3//consoleText) |
| :heavy_check_mark: | ef10aa06 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/3/) | 2023-04-23 16:18:28 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230423-161514-ef10aa-pr3397.zip) |
| :heavy_check_mark: | ef10aa06 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/3/) | 2023-04-23 16:19:31 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230423-161509-ef10aa-pr3397.zip) |
| :heavy_check_mark: | ef10aa06 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/3/) | 2023-04-23 16:19:56 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230423-161509-ef10aa-pr3397.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 611128db | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/4/) | 2023-04-23 17:37:04 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/4//consoleText) |
| :heavy_check_mark: | 611128db | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/4/) | 2023-04-23 17:38:16 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230423-173559-611128-pr3397.zip) |
| :heavy_check_mark: | 611128db | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/4/) | 2023-04-23 17:38:53 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230423-173559-611128-pr3397.zip) |
| :heavy_check_mark: | 611128db | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/4/) | 2023-04-23 17:40:32 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230423-173559-611128-pr3397.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 2ca63f49 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/5/) | 2023-04-25 15:29:06 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/5//consoleText) |
| :heavy_check_mark: | 2ca63f49 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/5/) | 2023-04-25 15:29:10 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230425-152630-2ca63f-pr3397.zip) |
| :heavy_check_mark: | 2ca63f49 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/5/) | 2023-04-25 15:29:23 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230425-152630-2ca63f-pr3397.zip) |
| :heavy_check_mark: | 2ca63f49 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/5/) | 2023-04-25 15:32:54 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230425-152630-2ca63f-pr3397.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 88c55385 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/6/) | 2023-04-25 16:11:29 | ~58 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/6//consoleText) |
| :heavy_check_mark: | 88c55385 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/6/) | 2023-04-25 16:13:03 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230425-161030-88c553-pr3397.zip) |
| :heavy_check_mark: | 88c55385 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/6/) | 2023-04-25 16:14:01 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230425-161030-88c553-pr3397.zip) |
| :heavy_check_mark: | 88c55385 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/6/) | 2023-04-25 16:14:12 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230425-161030-88c553-pr3397.aar) |
| | | | | | | |
| :heavy_multiplication_x: | dfca706b | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/7/) | 2023-04-26 11:16:33 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/7//consoleText) |
| :heavy_check_mark: | dfca706b | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/7/) | 2023-04-26 11:19:05 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230426-111426-dfca70-pr3397.aar) |
| :heavy_check_mark: | dfca706b | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/7/) | 2023-04-26 11:19:08 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230426-111430-dfca70-pr3397.zip) |
| :heavy_check_mark: | dfca706b | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/7/) | 2023-04-26 11:33:12 | ~18 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230426-111430-dfca70-pr3397.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 19e812d5 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/8/) | 2023-04-26 12:45:45 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/8//consoleText) |
| :heavy_check_mark: | 19e812d5 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/8/) | 2023-04-26 12:45:49 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230426-124324-19e812-pr3397.zip) |
| :heavy_check_mark: | 19e812d5 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/8/) | 2023-04-26 12:47:17 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230426-124322-19e812-pr3397.aar) |
| :heavy_check_mark: | 19e812d5 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/8/) | 2023-04-26 12:56:39 | ~13 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230426-124323-19e812-pr3397.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 28481d97 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/9/) | 2023-04-26 13:02:37 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/9//consoleText) |
| :heavy_check_mark: | 28481d97 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/9/) | 2023-04-26 13:02:58 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230426-125959-28481d-pr3397.zip) |
| :heavy_check_mark: | 28481d97 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/9/) | 2023-04-26 13:03:18 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230426-125955-28481d-pr3397.zip) |
| :heavy_check_mark: | 28481d97 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/9/) | 2023-04-26 13:04:01 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230426-125955-28481d-pr3397.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 4e28ed24 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/10/) | 2023-04-26 14:01:25 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/10//consoleText) |
| :heavy_check_mark: | 4e28ed24 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/10/) | 2023-04-26 14:02:30 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230426-140016-4e28ed-pr3397.zip) |
| :heavy_check_mark: | 4e28ed24 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/10/) | 2023-04-26 14:03:19 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230426-140016-4e28ed-pr3397.zip) |
| :heavy_check_mark: | 4e28ed24 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/10/) | 2023-04-26 14:03:57 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230426-140016-4e28ed-pr3397.aar) |
| :heavy_multiplication_x: | 4e28ed24 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/11/) | 2023-04-26 14:14:53 | ~9 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 5326a2d4 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/11/) | 2023-04-28 15:00:08 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230428-145750-5326a2-pr3397.zip) |
| :heavy_check_mark: | 5326a2d4 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/11/) | 2023-04-28 15:02:05 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230428-145750-5326a2-pr3397.aar) |
| :heavy_check_mark: | 5326a2d4 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/11/) | 2023-04-28 15:02:08 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230428-145750-5326a2-pr3397.zip) |
| :heavy_multiplication_x: | 5326a2d4 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/12/) | 2023-04-28 15:20:23 | ~22 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a119d83f | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/12/) | 2023-05-01 10:35:14 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230501-103303-a119d8-pr3397.zip) |
| :heavy_check_mark: | a119d83f | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/12/) | 2023-05-01 10:36:00 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230501-103303-a119d8-pr3397.zip) |
| :heavy_check_mark: | a119d83f | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/12/) | 2023-05-01 10:36:49 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230501-103303-a119d8-pr3397.aar) |
| :heavy_multiplication_x: | a119d83f | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/13/) | 2023-05-01 10:41:09 | ~8 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/13//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d11f60df | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/13/) | 2023-05-02 10:22:00 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230502-101937-d11f60-pr3397.zip) |
| :heavy_check_mark: | d11f60df | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/13/) | 2023-05-02 10:23:00 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230502-101937-d11f60-pr3397.zip) |
| :heavy_check_mark: | d11f60df | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/13/) | 2023-05-02 10:23:37 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230502-101933-d11f60-pr3397.aar) |
| :heavy_check_mark: | d11f60df | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/14/) | 2023-05-02 10:31:36 | ~11 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/14//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e179d34a | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/14/) | 2023-05-04 10:05:53 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230504-100208-e179d3-pr3397.zip) |
| :heavy_check_mark: | e179d34a | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/14/) | 2023-05-04 10:07:27 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230504-100208-e179d3-pr3397.zip) |
| :heavy_check_mark: | e179d34a | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/14/) | 2023-05-04 10:09:38 | ~7 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230504-100208-e179d3-pr3397.aar) |
| :heavy_check_mark: | e179d34a | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/15/) | 2023-05-04 10:16:04 | ~13 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/15//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 0631796e | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/16/) | 2023-05-04 16:45:42 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/16//consoleText) |
| :heavy_check_mark: | 0631796e | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/15/) | 2023-05-04 16:46:04 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230504-164341-063179-pr3397.zip) |
| :heavy_check_mark: | 0631796e | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/15/) | 2023-05-04 16:46:48 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230504-164341-063179-pr3397.zip) |
| :heavy_check_mark: | 0631796e | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/15/) | 2023-05-04 16:47:57 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230504-164341-063179-pr3397.aar) |
| | | | | | | |
| :heavy_check_mark: | 00873371 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/16/) | 2023-05-04 17:17:44 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230504-171546-008733-pr3397.zip) |
| :heavy_multiplication_x: | 00873371 | [#17](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/17/) | 2023-05-04 17:18:06 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/17//consoleText) |
| :heavy_check_mark: | 00873371 | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/16/) | 2023-05-04 17:18:49 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230504-171546-008733-pr3397.zip) |
| :heavy_check_mark: | 00873371 | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/16/) | 2023-05-04 17:19:11 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230504-171546-008733-pr3397.aar) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 40f3a150 | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/17/) | 2023-05-05 13:50:50 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230505-134816-40f3a1-pr3397.zip) |
| :heavy_check_mark: | 40f3a150 | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/17/) | 2023-05-05 13:53:13 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230505-134816-40f3a1-pr3397.aar) |
| :heavy_multiplication_x: | 40f3a150 | [#18](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/18/) | 2023-05-05 14:15:57 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/18//consoleText) |
| :heavy_check_mark: | 40f3a150 | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/17/) | 2023-05-05 14:23:02 | ~34 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230505-134816-40f3a1-pr3397.zip) |
| | | | | | | |
| :heavy_check_mark: | b83f180f | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3397/18/) | 2023-05-08 13:39:38 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230508-133645-b83f18-pr3397.zip) |
| :heavy_check_mark: | b83f180f | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3397/18/) | 2023-05-08 13:40:41 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230508-133645-b83f18-pr3397.aar) |
| :heavy_check_mark: | b83f180f | [#18](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3397/18/) | 2023-05-08 13:41:25 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230508-133645-b83f18-pr3397.zip) |
| :heavy_multiplication_x: | b83f180f | [#19](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/19/) | 2023-05-08 14:00:08 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/19//consoleText) |
| :heavy_multiplication_x: | b83f180f | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/20/) | 2023-05-08 15:51:20 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/20//consoleText) |
| :heavy_multiplication_x: | b83f180f | [#21](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/21/) | 2023-05-08 16:17:01 | ~20 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/21//consoleText) |
| :heavy_multiplication_x: | b83f180f | [#22](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/22/) | 2023-05-08 16:39:52 | ~20 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/22//consoleText) |
| :heavy_check_mark: | b83f180f | [#23](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/23/) | 2023-05-08 16:59:03 | ~10 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3397/23//consoleText) |

cammellos commented on protocol/messenger_chats.go:
```diff
@@ -76,10 +75,24 @@ func (m *Messenger) ChatsPreview() []*ChatPreview {
-				if chat.LastMessage.ContentType == protobuf.ChatMessage_TEXT_PLAIN {
 
+				if chat.LastMessage.ContentType == protobuf.ChatMessage_IMAGE {
+					chatPreview.ParsedText = chat.LastMessage.ParsedText
```
`GetSimplifiedText`?

vkjr commented:
@cammellos, could you please take a look at this PR again?
Per design in mobile app you should show a "message deleted" preview, when the last message is a deleted one. So I changed a few more places in the code to leave a deleted message in `chat.LastMessage` field. 
The only other logic where `chat.LastMessage` was used was the one with notifications, so I updated it a bit too.

cammellos commented:
@vkjr looks good, nice we can simplify, would you mind adding (or updating?) tests please?

vkjr committed:
More fields added to ChatPreview. LastMessage now can be a deleted message

vkjr commented:
@cammellos, thanks, I've updated tests and also found a bug that wasn't obvious before.
